.github/workflows/bot.yml

```diff
@@ -0,0 +1,150 @@
+name: CI Bot
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_review_comment:
+    types:
+      - created
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+
+env:
+  # This plugins is for anyone who can use it
+  PLUGINS: |-
+    assign
+    auto-cc
+    cc
+    label-bug
+    label-documentation
+    label-enhancement
+    label-question
+
+  # This plugins is for organization member or repository member
+  MEMBERS_PLUGINS: |-
+    label-duplicate
+    label-good-first-issue
+    label-help-wanted
+    label-invalid
+    label-kind
+    label-wontfix
+    label-lgtm
+    lifecycle
+    retitle
+
+  # This plugins is for in the REVIEWERS environment variable
+  REVIEWERS_PLUGINS: |-
+
+  # This plugins is for in the APPROVERS environment variable
+  APPROVERS_PLUGINS: |-
+    label
+    merge
```

Comment:
We may not need to use command to merge for now

Reply:
Done